Source/WebCore/page/EventHandler.cpp

```diff
@@ -1728,11 +1722,11 @@ void EventHandler::cancelAutoHideCursorTimer()
 
 void EventHandler::autoHideCursorTimerFired()
 {
-    auto* view = m_frame.view();
+    auto* view = m_frame->view();
```

Comment:
Use RefPtr?

Reply:
It only called `view->isActive()` so it seemed excessive?